iroh/src/endpoint.rs

```diff
@@ -23,31 +23,23 @@ use n0_watcher::Watcher;
 use tracing::{debug, instrument, trace, warn};
 use url::Url;
 
-use self::hooks::EndpointHooksList;
```

Comment:
The import ordering was off, and when I changed a line, `cargo make format` picked it up again. The only actual change here is that I added the re-exports from the new `remote_info` module.